### ISSUE TITLE
FR: Reports to show people in an age range

### ISSUE THREAD

**Issue by Gibbz** [tbar0970/jethro-pmm#122]
It would be really handy instead of having the age group drop down box to instead have a birthday field (or as well as?). 

As a church we would like to send out birthday cards + gifts for peoples birthdays.
It would be also handy if a reminder email to the Pastor went out once a week showing the next weeks birthdays. So that the gifts and cards can be organized in advance.

Also this would be handy for example doing a mail out to all people of certain age etc.

**Comment by tbar0970** [comment]
Yep.  You can set up a date field to do this - see System Admin > Date types.
Then you can run a report to show people with birthdays in the next 30 days, etc etc.
(In v.2.14 onwards these will be called "custom fields").

**Comment by tbar0970** [comment]
Email reminders would be a new feature.  It'd be good to think about how to do this generically.
Eg for people with [date field X] within the next [ Y ] days send a list to [ email addresses ].

**Comment by Gibbz** [comment]
Oh cool that is handy, I didn't see that option there in the reports! We have been using a custom field for birthdays.

Is there anyway to do the by age listing easily?

**Comment by tbar0970** [comment]
> Is there anyway to do the by age listing easily?
You mean listing people in order of birth date?  You can use a report for that too - see the "sort the results by..." section.

> Edit: there is an issue/bug where each time i edit the report, it continues to add new date fields even though i'm leaving them blank!
Can you post a screenshot?  This bit has been reworked in the upcoming new version so the bug may go away anyway.

**Comment by Gibbz** [comment]
I mean say list all people ages 30-40.

Screenshot of the issue:
![birthdays](https://cloud.githubusercontent.com/assets/650506/12804904/1771a046-cb47-11e5-9b8f-d3afb1ee13e4.jpeg)

**Comment by tbar0970** [comment]
OK, yeah, that will go away in v2.14.

Re people aged 30-40:  You can use a rule in a report "that have birthday exact value within the period 1 Jan 1976 - 1 Jan 1986.  I admit this would need adjusting as time went on.

**Comment by Gibbz** [comment]
Ok if your wanting to make it generic, how about having an option for example:
current date - xx years
Having to adjust it each year is a bit of a pain, but i guess it will work for now  :+1:

**Comment by tbar0970** [comment]
So what we really need is, in a report, a filter for 
"has [ birth date] between 30 and 40 years before today"

**Comment by tbar0970** [comment]
Resolved in #1199
